Scripts/Timer.cs

```diff
@@ -171,7 +171,7 @@ private void AddToUpdateQueue()
 
 		private void RemoveFromUpdateQueue()
 		{
-			if (!isUpdating)
+			if (isUpdating)
```

Comment:
So what we are saying is that everytime a timer stops, we don't unsubscribe from the update. So every single timer that ever started is still receiving updates. In every project! That's where all your memory leaks come from. Millions of subscribed timers!

Reply:
It is literally never been removed.

As long as you used this Timer in your project for the past 3 years... 👍 👍 👍

Reply:
I just saw this and you guys got me thinking I had someone to blame for a bug I'm dealing with, but unfortunately for me that `!isUpdating` check only got added in the previous commit a few weeks ago. So not 3 years of memory leaks solved I'm afraid!